src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -130,11 +159,20 @@ const CanonicalMostRead = ({ endpoint, maxTwoColumns }) => {
         </MostReadList>
       </MarginWrapper>
     </section>
-  ) : null;
+  );
+
+  return constrainMaxWidth ? (
+    <MaxWidthWrapper>
```

Comment:
Instead of wrapping this section in a div, could we not pass in the styles from `MaxWidthWrapper` onto the MostReadSection?

e.g. ```const MaxWidthWrapper = styled.div``` becomes ```const maxWidthStyles = css```
and then we can do this:
```
const ConstrainedMostReadSection = styled(MostReadSection)`
  ${maxWidthStyles}
`;
```

Reply:
Made this update and it's looking a good bit cleaner! Removed the need to define `MostRead` inside of `CanonicalMostRead` and can instead now `return`.